dev-texlive/texlive-latexextra/texlive-latexextra-2023_p69752-r2.ebuild

```diff
@@ -3947,7 +3947,7 @@ TEXLIVE_MODULE_SRC_CONTENTS="
 	zref-vario.source.r68846
 "
 
-inherit texlive-module
+inherit texlive-module optfeature
```

Comment:
The inherits should be sorted alphabetically.

Reply:
Hi @Flowdalic 
My apologies; I was not aware of that. I  changed it in the lastest commit